ubuntu/standard/Dockerfile.template

```diff
@@ -54,6 +54,15 @@ ENV GOROOT_1_9_X64=/usr/local/go1.9.4 \
     GOROOT=/usr/local/go1.10
 ENV PATH $PATH:$GOROOT/bin
 
+# Install HHVM
+RUN apt-get update \
+ && apt-get install software-properties-common apt-transport-https \
```

Comment:
You should pass -y to ensure there is no prompt. Also I think these packages are installed already by this point so it may turn into a no-op.

Reply:
I've verified that this line can be removed.